### PR TITLE
Task/User entity

### PR DESCRIPTION
**Pull request by vikkushnir**

### CODE REVIEW

File: src/main/java/com/ua/accommodation/model/Role.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false, unique = true)
+    private RoleName name;
+    @Column(nullable = false)
+    private boolean isDeleted;
+
+    public enum RoleName {
+        ADMIN,
+        USER
+
```
Comment (Nikname2303) [inline src/main/java/com/ua/accommodation/model/Role.java:35]: delete unnecessary string

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
   - include:
       file: db/changelog/changes/create-table-payments.yaml
+  - include:
+      file: db/changelog/changes/create-user-table.yml
+  - include:
+      file: db/changelog/changes/create-roles-table.yml
+  - include:
+      file: db/changelog/changes/create-users-roles-table.yml
+  - include:
+      file: db/changelog/changes/insert-users.yml
```
Comment (Nikname2303) [inline src/main/resources/db/changelog/db.changelog-master.yaml:11]: add empty line

**Review by Nikname2303 — CHANGES_REQUESTED** [review]
add empty line

**Review by VolandevlodD — APPROVED** [review]
LGTM

**Review by Nikname2303 — APPROVED** [review]

**Pull request merged**